estudios_socioeconomicos/models.py

```diff
@@ -47,6 +50,30 @@ def __str__(self):
                                 status=self.status)
 
 
+@receiver(post_save, sender=Estudio)
+def create_answers_for_study(sender, instance=None, created=False, **kwargs):
+    """ Signal for creating all answers for all questions on a new study.
+
+    This triggers creates all answer objects for all existing questions on a new
+    study. Since we are dealing with de-normalized data for the questions stored
+    in the database, we want to populate all the answers to query them and display
+    them to the user.
+
+    Parameters:
+    -----------
+      instance : estudios_socioeconomicos.models.Estudio
+          The instance of the object whose creation triggered the signal. In this case a
+          Estudio.
+      created : BooleanField
+          A value indicating if this instance is being created for the first time. Or if set
+          to false if it is being edited.
+    """
+    if created:
```

Comment:
So as the Capturistas put more different answers to a question the DB will be populating itself here?